### PR TITLE
let kops build take in GCS_LOCATION

### PR DESCRIPTION
**Pull request by krzyzacy**
ref current run: https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/kops/2987/pull-kops-e2e-kubernetes-aws-scenario/2/

/assign @chrislovecnm @zmerlynn @justinsb

### CODE REVIEW

File: scenarios/kubernetes_kops_aws.py [hunk 1]
```diff
@@ -124,11 +124,11 @@ def main(args):
         if not os.path.basename(workspace) == 'kops':
             raise ValueError(workspace)
         version = 'pull-' + check_output('git', 'describe', '--always').strip()
-        check('make', 'gcs-publish-ci', 'VERSION=%s' % version)
-        gcs = 'gs://kops-ci/pulls/pull-kops-e2e-kubernetes-aws-scenario'
+        gcs = 'gs://kops-ci/pulls/%s' % os.getenv('JOB_NAME', 'pull-kops-e2e-kubernetes-aws')
```
Comment (chrislovecnm) [inline scenarios/kubernetes_kops_aws.py]: Are we checking that JOB_NAME is set? Otherwise looks good.

Comment (krzyzacy) [inline scenarios/kubernetes_kops_aws.py]: if `JOB_NAME` is not set, it will use `pull-kops-e2e-kubernetes-aws` as the default.

Comment (chrislovecnm) [inline scenarios/kubernetes_kops_aws.py]: Long time since I read python :facepalm

**Comment by krzyzacy** [comment]
thanks :-)

**Pull request merged**